dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1464,12 +1476,75 @@
         }
       ]
     },
+    "EntitiesLROTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for an Entity Recognition LRO task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/EntitiesTaskParameters"
+        },
+        {
+          "$ref": "#/definitions/EntitiesUDEParameters"
+        }
+      ]
+    },
+    "EntitiesUDEParameters": {
```

Comment:
Acronyms that are not well-known are not allowed for the SDK. Consider `UserDefinedEntityParameters`.